pages/services/nifi/0.4.0-1.8.0/troubleshooting/debugging/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: layout.pug
+navigationTitle:  Debugging
+title: Debugging
+menuWeight: 101
+excerpt: Some common errors you might find
+featureMaturity:
+enterprise: false
+model: ../../../data.yml
+render: mustache
+---
+
+# Configuration update errors
+
+After a configuration change, the service may enter an unhealthy state. This commonly occurs when an invalid configuration change was made by the user. Certain configuration values may not be changed, or may not be decreased. To verify whether this is the case, check the service’s deploy plan for any errors.
+
+```shell
+dcos {{ model.serviceName }} --name={{ model.serviceName }}-dev plan show deploy
+```
+
+# Accessing Logs
+
+Logs for the scheduler and all service nodes can be viewed from the DC/OS web interface.
+
+    - Scheduler logs are useful for determining why a node isn’t being launched (this is under the purview of the Scheduler).
```

Comment:
Please reduce the indent on these bulleted lists. They are showing up as code blocks.

Reply:
done